bedrock/mozorg/templates/mozorg/home/includes/pocket-promo.html

```diff
@@ -0,0 +1,39 @@
+{#
+  This Source Code Form is subject to the terms of the Mozilla Public
+  License, v. 2.0. If a copy of the MPL was not distributed with this
+  file, You can obtain one at https://mozilla.org/MPL/2.0/.
+#}
+
+{% from "macros-protocol.html" import split with context %}
+
+{% call split(
+  block_class='mzp-l-split-reversed mzp-t-split-nospace mzp-l-split-center-on-sm-md t-pocket-promo',
+  theme_class='mzp-c-split-bg',
+  body_class=None,
+  media_class='mzp-l-split-h-center mzp-l-split-v-center',
+  media_after=False,
+  image=resp_img(
+          url='img/home/2023/pocket-promo/pocket-promo-text-900.jpg',
+          srcset={
+            'img/home/2023/pocket-promo/pocket-promo-text-1280.jpg': '1200w',
+            'img/home/2023/pocket-promo/pocket-promo-text-900.jpg': '900w',
+            'img/home/2023/pocket-promo/pocket-promo-text-700.jpg': '700w',
+            'img/home/2023/pocket-promo/pocket-promo-mobile-650.jpg': '650w',
+            'img/home/2023/pocket-promo/pocket-promo-mobile-520.jpg': '520w',
+          },
```

Comment:
[Needs a `sizes` declaration](https://bedrock.readthedocs.io/en/latest/coding.html#resp-img) otherwise it assumes the image takes up the full width of the screen.

Reply:
Based on the Figma file, they wanted the images to be full-width with smaller screens